flow360/component/simulation/models/volume_models.py

```diff
@@ -444,6 +444,12 @@ class Rotation(Flow360BaseModel):
         description="The parent rotating entity in a nested rotation case."
         + "The entity should be :class:`Cylinder` or :class:`GenericVolume` type.",
     )
+    noninertial_reference_frame_model: Optional[bool] = pd.Field(
+        None,
+        description="Flag to specify whether the noninertial referene frame model (MRF) is "
```

Comment:
`noninertial referene frame model (MRF)` Abbreviation?

Reply:
Removed the abbreviation. I had left it their for ease of understanding to the clients. Thanks.